src/integrations/env_canada.ts

```diff
@@ -70,7 +68,30 @@ export default class EnvironmentCanada implements MeteoalarmIntegration {
 			'Weather': MeteoalarmEventType.Unknown,
 			'Wind': MeteoalarmEventType.Wind,
 			'Winter storm': MeteoalarmEventType.SnowIce,
-			'Special Weather': MeteoalarmEventType.Unknown
+		// From: https://www.canada.ca/fr/environnement-changement-climatique/services/types-previsions-meteorologiques-utilisation/publiques/criteres-alertes-meteo.html	
+			'Poussée d’air arctique': MeteoalarmEventType.SnowIce,
+		// Same as English	'Blizzard': MeteoalarmEventType.SnowIce,
+			'Poudrerie': MeteoalarmEventType.SnowIce,
+			'Tempête de poussière': MeteoalarmEventType.Dust,
+			'Froid extrême': MeteoalarmEventType.LowTemperature,
+			'Refroidissement soudain': MeteoalarmEventType.SnowIce,
+			'Brouillard': MeteoalarmEventType.Fog,
+			'Bruine verglaçante': MeteoalarmEventType.SnowIce,
+			'Pluie verglaçante': MeteoalarmEventType.SnowIce,
+			'Gel': MeteoalarmEventType.SnowIce,
+			'Chaleur': MeteoalarmEventType.HighTemperature,
+			'Ouragan': MeteoalarmEventType.Hurricane,
+			'Pluie': MeteoalarmEventType.Rain,
+			'Orage violent': MeteoalarmEventType.Thunderstorms,
+			'Neige': MeteoalarmEventType.SnowIce,
+			'Bourrasques de neige': MeteoalarmEventType.SnowIce,
+			'Onde de tempête': MeteoalarmEventType.Thunderstorms,
+			'Tornade': MeteoalarmEventType.Tornado,
+			'Tempête tropicale': MeteoalarmEventType.Hurricane,
+		// Same as English	'Tsunami': MeteoalarmEventType.Tsunami,
+			'Météorologique': MeteoalarmEventType.Unknown,
+			'Vents': MeteoalarmEventType.Wind,
+			'Tempête hivernale': MeteoalarmEventType.SnowIce,
```

Comment:
I guess you should make this function to return `[key: [string, string]: MeteoalarmEventType]` and modify `getAlerts` to use that.

That would look something like
```js
['Arctic outflow', 'Poussée d’air arctique']: MeteoalarmEventType.SnowIce
```

Right now it's just mess